inc/saiqueue.h

```diff
@@ -96,6 +96,28 @@ typedef enum _sai_queue_pfc_continuous_deadlock_state_t
 
 } sai_queue_pfc_continuous_deadlock_state_t;
 
+/**
+ * @brief Enum defining queue actions in case of packet discard.
+ */
```

Comment:
don't we want to have this enum type and its use in the Buffer Profile ? Then this buffer profile will be associated with relevant Queues ony (where trimming is enabled)